src/App.vue

```diff
@@ -257,6 +258,14 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     return process.env.ABOUT_TEXT
   }
 
+  /** Track changes to alteration specific pieces. */
+  private get hasAlterationChanges (): boolean {
```

Comment:
Is this used anywhere?

Reply:
Yes, line 457 in the `goToDashboard` method.